src/cookbooks/new_ping.md

```diff
@@ -3,189 +3,247 @@
-Once you've sent some pings, refer to the following real-time analysis plugins
-to verify that your data is being processed:
-
-- `https://pipeline-cep.prod.mozaws.net/dashboard_output/graphs/analysis.moz_generic_error_monitor.<namespace>.html`
-- `https://pipeline-cep.prod.mozaws.net/dashboard_output/analysis.moz_generic_<namespace>_<doctype>_<docversion>.submissions.json`
-- `https://pipeline-cep.prod.mozaws.net/dashboard_output/analysis.moz_generic_<namespace>_<doctype>_<docversion>.errors.txt`
-
-If this first graph shows ingestion errors, you can view the corresponding
-error messages in the third link. Otherwise, you should be able to view the
-last ten processed submissions via the second link. You can also write your own
-custom real-time analysis plugins using this same infrastructure if you desire;
-use the above plugins as examples and see
-[here](realtime_analysis_plugin.md) for a more detailed explanation.
-
-If you encounter schema validation errors, you can fix your data or
-[submit another pull request](#submit-schema-to-mozilla-servicesmozilla-pipeline-schemas)
-to amend your schemas. Backwards-incompatible schema changes should generally
-be accompanied by an increment to `docversion`.
-
-Once you've established that your pings are flowing through the real-time
-system, verify that you can access the data from the downstream systems.
-
-### STMO
-In the Athena data source, a new table
-`<namespace>_<doctype>_parquet_<docversion>` will be created for your data. A
-convenience pointer `<namespace>_<doctype>_parquet` will also refer to the latest
-available `docversion` of the ping. The data is partitioned by
-`submission_date_s3` which is formatted as `%Y%m%d`, like `20180130`, and is
-generally updated hourly. Refer to the [STMO documentation](../tools/stmo.md)
-for general information about using Re:dash.
-
-This table may take up to a day to appear in the Athena source; if you still
-don't see a table for your new ping after 24 hours,
-[contact Data Operations](https://mana.mozilla.org/wiki/display/SVCOPS/Contacting+Data+Operations)
-so that they can investigate. Once the table is available, it should contain
-all the pings sent during that first day, regardless of how long it takes for
-the table to appear.
+## Deployment
+
+Schemas are automatically deployed once a day around 00:00 UTC, scheduled after the probe scraper in
+the following [Airflow
+DAG](https://github.com/mozilla/telemetry-airflow/blob/master/dags/probe_scraper.py). The latest
+schemas can be viewed at
+[`mozilla-pipeline-schemas/generated-schemas`](https://github.com/mozilla-services/mozilla-pipeline-schemas/tree/generated-schemas).
+
+## Start Sending Data
+
+Use the built-in Telemetry APIs when possible. A few examples are the [Gecko Telemetry
+APIs](https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/telemetry/collection/custom-pings.html),
+the [Android Telemetry APIs](https://github.com/mozilla-mobile/telemetry-android), or the [iOS
```

Comment:
I sort of think we shouldn't even include Android Telemetry, and note again that Android should use Glean, which doesn't require this.